asciidoc/volume2/gateways/tf2-ch-b-gateway-obx18-mapping.adoc

```diff
@@ -73,22 +51,22 @@ NOTE: <<ref_tbl_dec_obx18_vmd_mapping>> defines the mapping of the <<acronym_mdi
 
 |OBX-18/EI-1
 |Entity Identifier
-|
-|
+|pm:Vmd/ext:Extension+++<wbr/>+++/sdpi:EquipmentIdentifier
+|Universal Unique Identifier (UUID) without the prefix *"urn:uuid:"*.
 
 |OBX-18/EI-2
 |Namespace ID
 |
-|
+|This field is left empty.
 
 |OBX-18/EI-3
 |Universal ID
-|
-|
+|pm:Vmd/ext:Extension+++<wbr/>+++/sdpi:EquipmentIdentifier
+|Universal Unique Identifier (UUID) without the prefix *"urn:uuid:"*.
 
 |OBX-18/EI-4
 |Universal ID Type
 |
-|
+|Set to *"UUID"*

```

Comment:
Is OBX-18/EI-X required to be a UUID?

Reply:
In HL7, the Universal Type ID specifies that the set id is a UUID. There is no prefix such as urn:uuid.